### DISCUSSION TITLE
Syncing between devices - Question

### DISCUSSION THREAD

**Discussion by delinuxco** [keepassxreboot/keepassxc#6690] [Q&A] [answered]
So this is not a Feature request but it was the closest to my issue.

I wanted to be able to sync multiple computers including one inside a VM with KeepassXC, the KeeShare seemed overly complicate for my needs. So I had a thought of trying to use my existing syncthing setup to share the KeepassXC database between devices. Since synchting is designed to avoid MIM situations it seemed logical to me.

Well it actually works really well, I can't find any fault with do it this way. I install KeepassXC in the different devices and create a share in my home root directory specifically for the KeepassXC database. I do make sure to check "Automatically reload the database when modified externally" and it all just works.

My question, is this.... Are there any inherently bad security issues in doing this?

**Comment by droidmonkey** [comment, marked as answer]
This is exactly the way to do it. Glad it is working well for you. Converting this to a discussion.

**Comment by physkets** [comment]
I currently have it setup so that that the `.kdbx` database file is synced across devices using Syncthing. Should I be using a KeeShare'd file for this instead of the database file directly? If yes, what benefit does that provide?

**Reply by droidmonkey** [reply]
No, use the database file directly. KeeShare is only useful if you want to export a portion of your database to another file for someone else
